src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylish = sourceString
+    .split(';')
+    .filter((things) => things.trim() !== '');
+
+  const result = {};
+
+  for (const thing of stylish) {
```

Comment:
I would rather call it 'styleRule' or something instead of a 'thing' but it's not a big deal here